container/README.md

```diff
@@ -0,0 +1,110 @@
+# watson-conversation-slots-intro-kube
+
+This directory allows you to deploy the watson-conversation-slots-intro application into a container running on Bluemix. 
+
+# Steps
+
+## Create the Kubernetes cluster
```

Comment:
we usually provide both minikube and bluemix instructions.

Reply:
Can do, unless someone thinks otherwise.

Reply:
I'll do minikube instructions in a separate patch.